From: uazo <uazo@users.noreply.github.com>
Date: Wed, 11 Oct 2023 10:33:02 +0000
Subject: Temp remove support for shared workers

Due to bugid 1147281 and 1490268 temporarily remove
support for shared workers (active only in desktops)

Ref
https://bugs.chromium.org/p/chromium/issues/detail?id=1147281
https://bugs.chromium.org/p/chromium/issues/detail?id=1490268

Test with https://worker-playground.glitch.me/
---
 chrome/browser/chrome_content_browser_client.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -3150,6 +3150,7 @@ bool ChromeContentBrowserClient::AllowSharedWorker(
     content::BrowserContext* context,
     int render_process_id,
     int render_frame_id) {
+  if ((true)) return false;
   DCHECK_CURRENTLY_ON(BrowserThread::UI);
 
   // Check if cookies are allowed.
--
2.25.1
